elements/pl-checkbox/pl-checkbox.py

```diff
@@ -78,11 +83,14 @@ def prepare(element_html, data):
     if not (0 <= min_incorrect <= max_incorrect <= len_incorrect):
         raise ValueError('INTERNAL ERROR: incorrect number: (%d, %d, %d, %d)' % (min_incorrect, max_incorrect, len_incorrect, len_correct))
 
-    min_select = pl.get_integer_attrib(element, 'min-select', MIN_SELECT_DEFAULT)
-    max_select = pl.get_integer_attrib(element, 'max-select', number_answers)
+    if allow_blank == True:
```

Comment:
I think this is a little overcomplicated. I think this can be replaced by something like:
```python
min_select_default = MIN_SELECT_BLANK if allow_blank else MIN_SELECT_DEFAULT
min_select = pl.get_integer_attrib(element, 'min-select', min_select_default)
if min_select < min_select_default:
    raise ValueError("...")